scikeras/wrappers.py

```diff
@@ -250,6 +255,23 @@ def _fit_keras_model(self, X, y, sample_weight, warm_start, **kwargs):
         # order implies kwargs overwrites fit_args
         fit_args = {**fit_args, **kwargs}
 
+        if OS_IS_WINDOWS:
+            # see tensorflow/probability#886
+            if not isinstance(X, np.ndarray):  # list, tuple, etc.
+                X = [
+                    X_.astype(np.int64) if X_.dtype == np.int32 else X_
+                    for X_ in X
+                ]
+            else:
+                X = X.astype(np.int64) if X.dtype == np.int32 else X
+            if not isinstance(y, np.ndarray):  # list, tuple, etc.
+                y = [
+                    y_.astype(np.int64) if y_.dtype == np.int32 else y_
+                    for y_ in y
+                ]
+            else:
+                y = y.astype(np.int64) if y.dtype == np.int32 else y
+
```

Comment:
The windows failures are now passing with this hack... I think this is an actual bug in TF that they should fix, but I guess this will do for now...

Reply:
Unfortunately, I did not see one. It might be as easy as:

```python3
import numpy as np
from tensorflow.python.framework.constant_op import convert_to_eager_tensor


convert_to_eager_tensor(np.array([1], dtype=np.int32)
```